src/main/kotlin/com/model/ReservationDao.kt

```diff
@@ -0,0 +1,18 @@
+package com.model
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.query.Param
+import org.springframework.stereotype.Repository
+import java.time.LocalDateTime
+
+@Repository
+interface ReservationDao : JpaRepository<Reservation, Long> {
+
+    @Query("SELECT r FROM Reservation r WHERE CAR_LICENSE_PLATE = :licensePlate AND endTime = null")
```

Comment:
czy nie da się tego napisać bez `@Query`? czemu nie wybrałeśinnego rozwiązania?

Reply:
Da się używając samych nazw metod do query, i takie właśnie było moje początkowe założenie ale wsparcie podpowiedzi IntelliJa do robienia tego typu zapytań jeszcze nie ma w kotlinie. Więc aby sprawdzić składnie musiałem tworzyć interfejs Javowy. Więc postanowiłem po prostu skorzystać z Query, aczkolwiek to była jedna z kwestii otwartych dla mnie, których nie byłem pewien.